docs/_static/vdr/v1.yaml

```diff
@@ -362,4 +377,40 @@ components:
         type:
           description: The type of the key.
           example: "JsonWebKey2020"
-          type: string
\ No newline at end of file
+          type: string
+    DIDCreateRequest:
+      properties:
+        controllers:
+          description: |
+            List of DIDs that can control the new DID Document. If selfControl = true and controllers is not empty.
+            The newly generated DID will be added to the list of controllers.
+          type: array
+          items:
+            type: string
+            description: DID according to Nuts specification
+            example: "did:nuts:128903fjgfslcnmgpe84"
+        assertionMethod:
+          type: boolean
+          description: indicates if the generated key pair can be used for assertions. Defaults to true.
```

Comment:
It's unclear to me if `the generated key pair` refers to one pair per document used for multiple purposes or that every boolean set to true creates an extra key pair.

Reply:
a single, added it to the POST description.